source/plugin/project/gradle.rst

```diff
@@ -27,50 +23,54 @@ you have chosen before.**
 .. code-block:: groovy
 
     plugins {
-        id 'java'
-        id 'org.spongepowered.plugin' version '0.11.3'
+        'java-library'
+        id("org.spongepowered.gradle.plugin") version "2.0.0"
     }
 
-    // This may not be required, but has solved issues in the past
-    compileJava.options.encoding = 'UTF-8'
-
     // TODO: Change the following to match your information
     group = 'com.example'
     version = '1.0.0-SNAPSHOT'
-    description = 'Here lies an example plugin definition'
 
     repositories {
-        jcenter()
+        mavenCentral()
     }
 
-    dependencies {
-        compile 'org.spongepowered:spongeapi:8.0.0'
+    sponge {
+        apiVersion("8.0.0-SNAPSHOT")
+        license("CHANGEME")
+        loader {
+            name(PluginLoaders.JAVA_PLAIN)
+            version("1.0")
+        }
+        plugin("example") {
+            description("Just testing things...")
+            entrypoint("org.spongepowered.example.Example")
+        }
+    }
+
+    val javaTarget = 8 // Sponge targets a minimum of Java 8
```

Comment:
`val` here is a Kotlin keyword, but the rest of this script it groovy, so you wouldn't have `val` here (you _might_ choose to use `def`, but I probably wouldn't here).

Make sure that the examples are consistent in the language they are written in.

Reply:
Changed to kotlin